public/js/admin.js

```diff
@@ -25,29 +25,15 @@ async function getNotes() {
 }
 
 // eslint-disable-next-line
-function createNote() {
+async function createNote() {
     const note = document.querySelector('#add-a-note').value;
-    fetch('/note/create', {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        redirect: 'follow',
-        body: JSON.stringify({ note })
-    });
+    await postServer('/note/create',note);
```

Comment:
I'm not sure if you ran this through ESLint or Prettier or some other program that does styling, but there should be a space after the `,` towards the end of this line.  Further, I'd ask that you call this function `postToServer` so it's a little more semantic.

Reply:
Also, this is a good place to do `const result = await postToServer[...]` and check if `result` passed or failed.

Reply:
^ you don't need to do this, but it's good practice. Please, though, add that space after the comma :)

Reply:
Resolved